### DISCUSSION TITLE
如何更新单个节点的值

### DISCUSSION THREAD

**Discussion by cdRunnet** [hizzgdev/jsmind#627] [General]
Hello, I would like to know how to update the value of a single node. I saw a value in the document that can attach an extra data. I added other styles to a single node and when I want to update the node, I will update the data in the other styles
![企业微信截图_17254199184416](https://github.com/user-attachments/assets/26dfb550-bdd4-40eb-a770-420229d2240a)
![企业微信截图_17254199832501](https://github.com/user-attachments/assets/916c8237-ea2e-46cf-8731-d5567eea1757)

**Comment by hizzgdev** [comment]
https://hizzgdev.github.io/jsmind/docs/zh/3.operation.html

```javascript
let node = jm.get_node(node_id)

// update the extra data
node.data = {... new data }

// update topic
jm.update_node(node_id, topic)
```

Is that what you want to know?
